### PR TITLE
[MODULAR] Readds Peacekeeper lockers with animation

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

See title.

Also adds animations to the CC locker.

## Why It's Good For The Game

The current red sec lockers give me an aneurysm, and the new sec_haul lockers were only removed because they weren't animated, which they now are.

## Changelog
:cl:
add: Readds Peacekeeper lockers with animation
add: adds animation for cc locker
/:cl:

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
CONFLICTS
REEEE

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	icon = 'modular_skyrat/modules/sec_haul/icons/lockers/closet.dmi'
+
+/obj/structure/closet/secure_closet/armory1
+	icon = 'modular_skyrat/modules/sec_haul/icons/lockers/closet.dmi'
+
+/obj/structure/closet/secure_closet/armory2
+	icon = 'modular_skyrat/modules/sec_haul/icons/lockers/closet.dmi'
+
+/obj/structure/closet/secure_closet/armory3
+	icon = 'modular_skyrat/modules/sec_haul/icons/lockers/closet.dmi'
```
Comment (FlamingLily) [inline modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm]:
```suggestion
	icon = 'modular_skyrat/modules/sec_haul/icons/lockers/closet.dmi'
	
```

Comment (FlamingLily) [reply modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm]: dm gets angy w/ no newlines at end of file

**Review by FlamingLily — CHANGES_REQUESTED** [review]

**Comment by Gandalf2k15** [comment]
You will need to update the actual code for some lockers with the angry comment next to them.

**Pull request merged**